frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -98,14 +95,12 @@ action.GitMachete.BaseSlideInBranchBelowAction.notification.message.entry-is-roo
 action.GitMachete.BaseSlideInBranchBelowAction.notification.message.slide-in-under-itself-or-its-descendant=Cannot slide in the branch under itself or any of its descendants
 action.GitMachete.BaseSlideInBranchBelowAction.notification.mismatched-names=<b>Slide in canceled:</b> branch name ({0}) and new branch name ({1}) differ
 action.GitMachete.BaseSlideInBranchBelowAction.task-title=Sliding in
-action.GitMachete.BaseSlideInBranchBelowAction.text.current-branch=Slide _In Branch Below Current Branch...
 
 
 action.GitMachete.BaseSlideOutBranchAction.description=Slide out ''{0}''
 action.GitMachete.BaseSlideOutBranchAction.description.root.branch=Root branch ''{0}'' cannot be slid out
 action.GitMachete.BaseSlideOutBranchAction.notification.fail=Slide out of <b>{0}</b> failed
 action.GitMachete.BaseSlideOutBranchAction.notification.success=Branch <b>{0}</b> slid out
-action.GitMachete.BaseSlideOutBranchAction.text.current-branch=Slide _Out Current Branch
```

Comment:
While you're here... `action.GitMachete.ISyncToRemoteStatusDependentAction.description.disabled.undefined.sync-to-remote` and `action.GitMachete.PullCurrentBranchFastForwardOnlyBackgroundable.notification.title.all-files-are-up-to-date` look completely unused